### ISSUE TITLE
Reading line numbers with hidden text in Word

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#5966]
With NVDA's line numbering enabled in Document formatting, in Microsoft Word:

If there is hidden text which spans at least one line (ie a word hidden in the middle of a line won't trigger this):

- With formatting marks shown the hidden lines are counted and read (and visible so that's expected) and with formatting marks turned off the hidden lines are hidden and lines not counted.  That is consistent with the way Word displays line numbers.

- If there is a blank (visible) line after the hidden text, NVDA reads simply "blank" for that line and does not read a line number.

- If there isn't a blank line after the hidden text, as you move by line it reads fine, but if you move by character, NVDA reads a line number, the first letter, the next line number then the rest of the line.  EG:

I wrote three lines "The first line / The second hidden line / The third line"  I hid the second line.  If I read by line or up and down arrow it reads line 1, the first line, line 2 the third line.  If I move by character, it reads through the first line then linefeed, then line 2 T, then line 3 h then through the rest of "e third line".

**Comment by michaelDCurran** [comment]
I am finding this hard to reproduce. Can you please attach a word document already set up to exhibit this behavior? thanks.

**Comment by Qchristensen** [comment]
Here's a document I created (wrote all the lines then selected and set hidden the ones I indicated).  I noticed that moving by character seems to work properly here but the moving by line characteristics are there.

[line numbers.docx](https://github.com/nvaccess/nvda/files/329737/line.numbers.docx)

**Comment by bhavyashah** [comment]
@michaelDCurran Could you please take another shot and attempting to reproduce this issue using the test attachment provided by @Qchristensen in https://github.com/nvaccess/nvda/issues/5966#issuecomment-228014057?

**Comment by Adriani90** [comment]
I can reproduce this issue with the uploaded document in Word 2016 and 365. Thank you @Qchristensen for uploading this example. In Word 2010 however I cannot reproduce this issue.
Note: when UIA for MS Word is enabled, line numbers are not announced at all in Word 2016 and 365. cc: @michaelDCurran
